autoflake8/fix.py

```diff
@@ -472,7 +472,10 @@ def is_literal_or_name(value: bytes) -> bool:
     return re.match(rb"^\w+\s*$", value) is not None
 
 
-def useless_pass_line_numbers(source: bytes) -> Iterator[int]:
+def useless_pass_line_numbers(
+    source: bytes,
+    keep_pass_after_docstring=False,
```

Comment:
nit: add type annotation
```suggestion
    keep_pass_after_docstring: bool = False,
```